wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -1206,6 +1206,12 @@ class SwiftGenerator private constructor(
       if (typeName is DeclaredTypeName) {
         imports.add(typeName.moduleName)
       }
+      if (field.isRepeated && typeName is ParameterizedTypeName) {
+        val declaredTypeName = typeName.typeArguments[0]
+        if (declaredTypeName is DeclaredTypeName) {
```

Comment:
is there a better way to avoid this check again? Is `typeArguments[0]` always this type?

Also I seem not to handle maps but only arrays?

Reply:
Tests could help you confirm all of those

Reply:
Confirmed my theory. Fixed now with a test.